command.go

```diff
@@ -136,7 +137,8 @@ func NewCommand(commandInfo mesos.CommandInfo, env []string, options ...func(*ex
 	//		similar to how POSIX exec families launch processes (i.e.,
 	//		execlp(value, arguments(0), arguments(1), ...)).
 	cmd := exec.Command("sh", "-c", commandInfo.GetValue()) // #nosec
-	cmd.Env = combineExecutorAndTaskEnv(env, commandInfo.GetEnvironment())
+	removeExecutorConfigurationEnv()
```

Comment:
It is not safe to modify global state of the executor process. Let this function return the cleared slice (without modifying the process environment) and use it in the started command.